mqtt-launcher.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=mqtt-launcher
+After=network.target
+
+[Service]
+PermissionsStartOnly=true
+ExecStart=/bin/bash -c 'source /opt/mqtt-launcher/bin/activate; /opt/mqtt-launcher/mqtt-launcher.py'
```

Comment:
We'll leave it as is, but I think just doing this also activates the venv for the duration of the program:

```
ExecStart=/opt/mqtt-launcher/bin/mqtt-launcher.py
```

if our .py is in the venv bin directory. It'd be slightly more efficient.